jenkins/CVE-2017-1000353/README.md

```diff
@@ -1,51 +1,55 @@
-# Jenkins-CI 远程代码执行漏洞（CVE-2017-1000353）
+# Jenkins-CI remote code execute exploit (CVE-2017-1000353)
 
-## 原理
+[中文版本(Chinese version)](README.zh-cn.md)
 
-参考阅读 https://blogs.securiteam.com/index.php/archives/3171
+## Reference Links
 
-## 测试环境搭建
+https://blogs.securiteam.com/index.php/archives/3171
 
-测试环境编译及运行：
+## Vulnerability environment
+
+Build and run：
 
 ```
 docker-compose build
 docker-compose up -d
 ```
 
-首次运行时会要求填写初始密码，默认在`/var/jenkins_home/secrets/initialAdminPassword`文件中，使用`docker-compose exec jenkins cat /var/jenkins_home/secrets/initialAdminPassword`即可读取。
+We will be asked setup initial password when first time run, and it will be stored at `/var/jenkins_home/secrets/initialAdminPassword`, use 
```

Comment:
The default initial password can be found by:

`docker-compose exec jenkins cat /var/jenkins_home/secrets/initialAdminPassword`